assets/stylesheets/modules/_promotion.scss

```diff
@@ -0,0 +1,97 @@
+.promotion__landing-page {
+    section:nth-child(even) {
+        background-color: rgb(231, 231, 231);
+
+        .page-slice {
+            background-color: rgb(241, 241, 241);
+        }
+    }
+    .page-slice {
+        padding-bottom: 0.75rem;
+
+        .page-slice__header {
+            font-weight: bold;
+            white-space: nowrap;
+        }
+    }
+}
+
+.promotion__landing-page__section-head {
+    margin-bottom: $gs-baseline * 2;
+}
+
+.promotion__digipack {
+    float: right;
+    width: auto;
+}
+
+.promotion__digipack-shots {
```

Comment:
Class names are supposed to reflect the HTML structure. The hyphenated `digipack-shots` is where you should be using BEM child syntax instead.
`.digipack` on parent
`.digipack__shots` on child